lms/templates/dashboard.html

```diff
@@ -196,7 +196,8 @@
                 show_courseware_link = show_courseware_links_for.get(session_id, False)
                 cert_status = cert_statuses.get(session_id)
                 can_refund_entitlement = entitlement and entitlement.is_entitlement_refundable()
-                can_unenroll = (not cert_status) or cert_status.get('can_unenroll') if not unfulfilled_entitlement else False
+                partner_managed_enrollment = enrollment.mode == 'masters'
```

Comment:
I thought this is not the logic we want to use. Why don't we want to reach into `program_enrollments` app and check for an existing active `program_course_enrollment` record associated with this learner/course?

Reply:
yup I started down that path but was concerned a future-use of program_enrollment may not be 1:1 with enrollments created by a university.  We had a brief parking lot discussion at some point.  While keying off 'masters' may also run into a similar problem in the future I figured it was less likely. Doing it this way we aren't adding a bunch of additional server-side logic that would need to be refactored.